tests/unit/utils/test_sdb.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email:`Vernon Cole <vernondcole@gmail.com>`
+'''
+
+# Import Python Libs
+from __future__ import absolute_import
+import os
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.unit import TestCase, skipIf
+from tests.support.mock import (
+    NO_MOCK,
+    NO_MOCK_REASON
+)
+
+# Import Salt Libs
+import salt.utils.sdb as sdb
+import salt.exceptions
+
+TEMP_DATABASE_FILE = '/tmp/salt-tests-tmpdir/test_sdb.sqlite'
```

Comment:
This will fail on Windows systems.

Reply:
I had guessed that there is a global or an environment variable for this, but a scan of the entire code base using PyCharm did not find it.  I failed to guess the correct keywords, it seems.

Perhaps someone would be so kind as to tell me the secret?

Reply:
You can use

```
from tests.support.runtests import RUNTIME_VARS
TEMP_DATABASE_FILE = os.path.join(RUNTIME_VARS.TMP, 'test_sdb.sqlite')
```

Reply:
I like it.